### DISCUSSION TITLE
Usage of PrimeFaces Template in Open Source project

### DISCUSSION THREAD

**Discussion by swissclash79** [primefaces/community#1547] [PrimeFaces Templates] [answered]
Dear PrimeFaces team

Is it allowed to use a PrimeFaces template in an Open Source project? What about if it is packaged in a JAR and the project uses it as a library?

Thank you very much for your answer

Best regards
Christian

**Comment by melloware** [comment]
@swissclash79 this same question was asked on Discord let me see if I can find the answer...

**Reply by melloware** [reply, marked as answer]
Found it:
![image](https://github.com/primefaces/community/assets/4399574/1b370eae-c192-4efe-a2e2-59633bf5e96c)

**Reply by swissclash79** [reply]
It seems there is no version of the Sakai template for PrimeFaces, but nevertheless thank you very much!
